packages/lisk-transactions/src/1_second_signature_transaction.ts

```diff
@@ -208,7 +209,12 @@ export class SecondSignatureTransaction extends BaseTransaction {
 
 	protected undoAsset(store: StateStore): ReadonlyArray<TransactionError> {
 		const sender = store.account.get(this.senderId);
-		const { secondPublicKey, ...strippedSender } = sender;
+		const strippedSender = {
+			...sender,
+			secondPublicKey: undefined,
```

Comment:
In Lisk-core, on undo action, the secondPublicKey property is assigned with the value `null`. Therefore, I think we should keep it consistent.

Reply:
@SargeKhan  We have a TS lint rule that forbids null and favours undefined should I skip it in this case?

Reply:
Yeah, I guess we have to do it in this particular case.